source/manual/cookies-and-sessions-in-frontend-apps.html.md

```diff
@@ -4,29 +4,28 @@ parent: "/manual.html"
 layout: manual_layout
 section: CDN & Caching
 owner_slack: "@bevan"
```

Comment:
Should we change the owner to 2nd line or devs?

Reply:
Yup, changed it to #govuk-developers